.babelrc

```diff
@@ -1,7 +1,19 @@
 {
-    "presets": ["es2015", "flow"],
+    "presets": [
+        ["es2015", { "modules": false }],
```

Comment:
Is setting `modules: false` to reduce overhead from the default `"commons"` setting?

Reply:
Setting to `modules: false` enables the use of tree-shaking with webpack.